templates/rhdm78-authoring-ha.yaml

```diff
@@ -1125,6 +1125,10 @@ objects:
             value: "insecure-${APPLICATION_NAME}-kieserver"
           - name: KIE_SERVER_STARTUP_STRATEGY
             value: "OpenShiftStartupStrategy"
+          - name: KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED
+            value: "true"
+          - name: KIE_SERVER_CONTROLLER_OPENSHIFT_GLOBAL_DISCOVERY_ENABLED
```

Comment:
@desmax74 No need to change, "...GLOBAL_DISCOVERY...", parameter.

Reply:
Ok @rhtevan